frontend/src/scenes/paths/NewPaths.tsx

```diff
@@ -271,7 +271,10 @@ export function NewPaths({ dashboardItemId = null, color = 'white' }: PathsProps
 
         const svg = createCanvas(width, height)
         const sankey = createSankey(width, height)
-        const { nodes, links } = sankey(paths)
+        const { nodes, links } = sankey({
+            nodes: paths.nodes.map((d) => ({ ...d })),
```

Comment:
non-blocking

Should we add in `...paths` as well to ensure that any other attributes that were considered and part of paths are also considered by `sankey`.  I have no idea if that's a likely scenario

Reply:
Reading the code in d3-sankey I'm pretty sure it's only node and links that are used by sankey

(We also explode paths the same way in the Old `Paths.tsx`)

Reply:
sankey code here https://github.com/d3/d3-sankey/blob/master/src/sankey.js

Reply:
and our older usage here https://github.com/PostHog/posthog/blob/master/frontend/src/scenes/paths/Paths.jsx#L139